public/app/core/components/NestedFolderPicker/NestedFolderList.tsx

```diff
@@ -68,39 +80,29 @@ interface RowProps {
 }
 
 function Row({ index, style: virtualStyles, data }: RowProps) {
-  const { items, foldersAreOpenable, selectedFolder, onFolderClick, onSelectionChange } = data;
-  const { item, isOpen, level } = items[index];
+  const { items, focusedItemIndex, foldersAreOpenable, selectedFolder, onFolderExpand, onFolderSelect } = data;
+  const { item, isOpen, level, parentUID } = items[index];
+  const rowRef = useRef<HTMLDivElement>(null);
+  const labelId = useId();
+  const children = items.filter((i) => i.parentUID === item.uid).map((i) => i.item.uid);
+  const posInSet = items.filter((i) => i.parentUID === parentUID).findIndex((i) => i.item.uid === item.uid) + 1;
```

Comment:
don't think we'll need to do this once we share state from redux -> we can use the `childrenByParentUID` selector